src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function get-product-list-width ($cards-in-line) {
+  @return $cards-in-line * ($card-width + $product-list-card-gap);
+}
```

Comment:
You can make it without function. Scss has an inbuild calc opportunity. 

Just create a variable with the similar formula, and there is no need to take a parameter of $cards-in-line like a variable, there always should be 4 cards.

Reply:
okey. replaced it.